torchgeo/trainers/segmentation.py

```diff
@@ -111,8 +104,13 @@ def configure_losses(self) -> None:
                 ignore_index=ignore_value, weight=self.hparams["class_weights"]
             )
         elif loss == "jaccard":
+            # JaccardLoss requires a list of classes to use instead of a class index to ignore.
+            class_set = [
```

Comment:
```suggestion
            classes = [
```
Not a set. I actually liked your previous solution better because it didn't involve any for-loops in Python.

Reply:
You gotta pick one

Reply:
I picked the previous solution, just with a list instead of a set 😆